nebula/ui/components/VPNServerCountry.qml

```diff
@@ -207,7 +207,8 @@ VPNClickableRow {
                         rightMargin: VPNTheme.theme.hSpacing
                         verticalCenter: parent.verticalCenter
                     }
-                    score: showConnectionScores ? modelData.connectionScore : (isAvailable ? VPNServerCountryModel.NoData : VPNServerCountryModel.Unavailable)
+                    score: (focusScope.currentServer.whichHop !== "multiHopExitServer") ? modelData.connectionScore
```

Comment:
Hmm, perhaps this would benefit from a comment to explain things better. The connection scores are trying to convey an idea of how good a location is when the user is browsing the server list, and it just so happens that the `modelData.connectionScore` property is valid for both the single-hop and multihop entry server selections, because we are representing the expected connection quality between the client and that chosen server.

It is only for the multihop exit where we need a special case, because we need to show the expected link performance between the entry and exit servers, and this has nothing to do with the user's location.

My preference was to put the common case first and then the multihop exit case second, but I can flip that around if you think it's better that way.

Reply:
> My preference was to put the common case first and then the multihop exit case second, but I can flip that around if you think it's better that way.

This makes sense. My comment was poorly worded. I think I would have better said: I find the current code a bit hard to read; can we think of ways of improving the readability? One though is keeping the test condition as a variable calculated elsewhere.  Second thought is that I find it easier to read non-negative boolean tests...but this may well just be me.